Assistant heirloom adjustments

Rukofamicom opened the pull request:
<!-- Write **BELOW** The Headers and **ABOVE** The comments else it may not be viewable. -->
<!-- You can view Contributing.MD for a detailed description of the pull request process. -->

## About The Pull Request
Fixed the visual bug with fingerless insulated gloves (both heirloom and normal) which caused them to appear white when equipped. Additionally, the assistant's heirloom toolbox is now rusted shut, so that it properly fits the negative heirloom trait.

<!-- Describe The Pull Request. Please be sure every change is documented or this can delay review and even discourage maintainers from merging your PR! -->

## Why It's Good For The Game
Gloves that magically turn white when equipped are a little bit off, and the family Heirloom is supposed to be a negative trait. Having your heirloom give you extra starting inventory space is not a negative thing by any means - there is no opportunity cost for carrying around extra inventory space throughout the round.

<!-- Please add a short description of why you think these changes would benefit the game. If you can't justify it in words, it might not be worth adding.
-->

## Testing

Yellow fingerless gloves equipped
![image](https://user-images.githubusercontent.com/9547572/167258245-de417b8b-0130-47aa-8e57-deb248187ed4.png)

Toolbox description and inability to be opened. 
![image](https://user-images.githubusercontent.com/9547572/167258286-52abf366-f65e-40e3-a6ea-1f0d32fe0cc0.png)

## Changelog
:cl:
balance: Heirloom toolboxes are now rusted shut and may no longer be used as extra storage space. 
fix: Fixed fingerless insulated gloves equipment sprite, they are now appropriately yellow instead of white.
/:cl:

<!-- Both :cl:'s are required for the changelog to work! You can put your name to the right of the first :cl: if you want to overwrite your GitHub username as author ingame. -->
<!-- You can use multiple of the same prefix (they're only used for the icon ingame) and delete the unneeded ones. Despite some of the tags, changelogs should generally represent how a player might be affected by the changes rather than a summary of the PR's contents. -->

Rukofamicom committed:
Fingerless insuls visual bug

Fingerless insuls, both the normal and the heirloom variety, now appear yellow when worn instead of white.

Rukofamicom committed:
Heirloom gas mask

No more heirloom toolbox

BrumalLake commented:
it isn't backpack sized and is identical to a cardboard box or paper bag

zeskorion commented:
to be fair, the toolbox is identical to an internals box
the only thing differentiating it is the fact that more people figured out you can put tools in boxes

Rukofamicom committed:
Fixes assistant objective

Rukofamicom commented:
And it's still a wholly beneficial effect compared to what familiy heirloom is supposed to be - a negative trait that either wastes some of your inventory space or constantly drains your mood. 

Someone in the discord server is working to convince me that the gas mask is still pretty useful by comparison to other heirlooms and I'm fairly inclined to agree.

Rukofamicom committed:
Revert "Fixes assistant objective"

This reverts commit eb255fc0bb93f5ff01a7c76734dae507d34653c7.

Rukofamicom committed:
Revert "Heirloom gas mask"

This reverts commit 595b2ec30003c230c8966088bc887548acbb9673.

Rukofamicom committed:
Back to a toolbox

Toolbox is now rusted shut so that it consumes inventory space instead of granting extra space, as an heirloom is intended to do.

Rukofamicom commented:
Rather than deciding between making a useless version of the gas mask or removing the assistant objective to acquire a gas mask, I just went back to the toolbox and made it a proper heirloom instead of a desirable item.

Rukofamicom commented:
Fixing tags here too. Apologies for my misuse of tweak instead of labeling as a balance change. 

While I don't consider it to have ever been intended for this negative trait to be used as a positive, it is still a rebalancing of something that has existed for a long time.

PowerfulBacon commented on code/datums/traits/negative.dm:
```diff
@@ -138,7 +138,7 @@
 				heirloom_type = pick(/obj/item/toy/windupToolbox, /obj/item/reagent_containers/food/drinks/bottle/holywater)
 			if("Assistant")
-				heirloom_type = pick(/obj/item/storage/toolbox/mechanical/old/heirloom, /obj/item/clothing/gloves/cut/heirloom)
+				heirloom_type = pick(/obj/item/heirloomtoolbox)//, /obj/item/clothing/gloves/cut/heirloom)
```
(Should probably choose the suggestion below)
```suggestion
				heirloom_type = /obj/item/heirloomtoolbox
```

PowerfulBacon replied:
Why remove the gloves though?

Rukofamicom replied:
Fuck, the glove removal was totally by accident.  It's commented out because I intended to put it back after testing lmao.

PowerfulBacon commented on code/datums/traits/negative.dm:
```diff
@@ -138,7 +138,7 @@
 				heirloom_type = pick(/obj/item/toy/windupToolbox, /obj/item/reagent_containers/food/drinks/bottle/holywater)
 			if("Assistant")
-				heirloom_type = pick(/obj/item/storage/toolbox/mechanical/old/heirloom, /obj/item/clothing/gloves/cut/heirloom)
+				heirloom_type = pick(/obj/item/heirloomtoolbox)//, /obj/item/clothing/gloves/cut/heirloom)
```
```suggestion
				heirloom_type = pick(/obj/item/heirloomtoolbox, /obj/item/clothing/gloves/cut/heirloom)
```

Rukofamicom committed:
Debugging comment fucking up me PR

Co-authored-by: PowerfulBacon <26465327+PowerfulBacon@users.noreply.github.com>

Rukofamicom committed:
Update toolbox.dm

Dallen999 commented:
A free cardboard box, Nerf fast.

Rukofamicom commented:
Better than free since it both provides a permanent (so long as you keep that useful box) positive mood boost and also enables the use of a positive quirk. 

Negative traits should be negative.
